code/modules/mob/living/carbon/human/species/necromorph/spitter.dm

```diff
@@ -185,7 +185,7 @@ Best used for harassment, skirmishing and initiating fights from afar against un
 	damage = 15
 	damage_type = BURN
 	nodamage = 0
-	nofire = TRUE
+	fire_volume = 0
```

Comment:
"temperature" not fire_volume

there might already be a var with that name, and if so just use it

Reply:
I've made var/fire_temp for temprature already